package.json

```diff
@@ -105,7 +105,7 @@
     "@babel/core": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
     "@eslint/core": "^0.4.0",
-    "@eslint/json": "^0.3.0",
+    "@eslint/json": "^0.4.0",
```

Comment:
This doesn't work, I'm getting the following error:

```js
$ npx eslint .
(node:11776) ESLintIgnoreWarning: The ".eslintignore" file is no longer supported. Switch to using the "ignores" property in "eslint.config.js": https://eslint.org/docs/latest/use/configure/migration-guide#ignoring-files
(Use `node --trace-warnings ...` to show where the warning was created)

Oops! Something went wrong! :(

ESLint: 9.9.1

TypeError: Cannot read properties of undefined (reading 'recommended')
    at Object.<anonymous> (C:\projects\eslint\eslint.config.js:185:25)
    at Module._compile (node:internal/modules/cjs/loader:1369:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Module._load (node:internal/modules/cjs/loader:1022:12)
    at cjsLoader (node:internal/modules/esm/translators:366:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:315:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
    at async loadFlatConfigFile (C:\projects\eslint\lib\eslint\eslint.js:390:21)
```

Reply:
Here's how the CJS export looks like now:

```js
exports.JSONLanguage = JSONLanguage;
exports.JSONSourceCode = JSONSourceCode;
exports.default = plugin;
```

Seems that after adding named exports in https://github.com/eslint/json/commit/ad729f0c60d42a84b2c87da52a6d2456b5211b48, rollup now exports the plugin as `default` in CJS, so we'd need to do `const json = require("@eslint/json").default;` instead of `const json = require("@eslint/json");`. @nzakas was this change intentional?

Reply:
it was intentional to export `JSONLanguage` and `JSONSourceCode`. I think this is just what Rollup does when it has multiple exports plus a default export in CJS. We should update the README to reflect that.

Reply:
Okay, then closing this in favor of https://github.com/eslint/eslint/pull/18829.